iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/AbstractIdentifiableAdderAdapter.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.mergingview;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.IdentifiableAdder;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+abstract class AbstractIdentifiableAdderAdapter<I extends IdentifiableAdder<I>> extends AbstractAdapter<I> implements IdentifiableAdder<I>  {
+
+    private String id;
```

Comment:
I don't understand why this variable is needed: can't you use the delegate?

Reply:
The delegate does not give acces to getter of "id" value.
Only method setId is accessible.
I have no choice to have a private field to keep in memory "id" value